### PR TITLE
View Friends Profile

### LINKED ISSUE #247: Create UI for View Friend Profile
refer to figma for implementation : https://www.figma.com/file/yCDFrt0sOYFhXlYlWp8sZT/Event-Radar-App?type=design&node-id=56-9&mode=design&t=2kB2TIeRwBaFglSk-0

# Description 

Create the UI to display profile information of a user's friend. (refer to the blueprint to know which information should be displayed)
The screen has a back button and a FAB message button that open the private chat between the user and its friend.

### PR DESCRIPTION
**Pull request by nikhen-s**
Related to #247 + #246 => Closes #247 + #246
## Objective
- Create a View Friends Profile UI + ViewModel

## Changes
- Added a "Bio" field for each user, and this is shown when you open a friend's profile & Refactored Code Accordingly
- Friend's Profile can be navigated through Contacts list
- Changed bottomNavigation font size to 11 as temporary fix for "Messages" text overflow bug.

## To Do after #241 is done
- After clicking on Friend's Profile in the Private Chat top bar, it should lead to the View for the Friend's Profile.

### CODE REVIEW

File: app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.github.se.eventradar.R
+
+@Composable
+fun UserProfileImage(profilePicLink: String, name: String, modifier: Modifier) {
```
Comment (christinehahaa) [inline app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt:18]: Nit: There is a similar function in ReusableComponents.kt. No code duplications on CI so not a priority to change.

**Review by christinehahaa — CHANGES_REQUESTED** [review]
Nik, this looks awesome. I texted on machine. No bugs (yay!). 

One tiny comment, can the name display both the first and last name?

File: app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt (hunk 1 above)
Comment (nikhen-s) [reply app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt:18]: There is a similar one, but the size and shape was different so I decided to create this one :)

Comment (christinehahaa) [reply app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt:18]: Gotcha!

**Review by christinehahaa — APPROVED** [review]
Changes LGTM!

**Pull request merged**